src/network/networkd-link.c

```diff
@@ -1607,10 +1621,13 @@ static int link_up(Link *link) {
                 if (r < 0)
                         return log_link_error_errno(link, r, "Could not open AF_INET6 container: %m");
 
-                ipv6ll_mode = link_ipv6ll_enabled(link) ? IN6_ADDR_GEN_MODE_EUI64 : IN6_ADDR_GEN_MODE_NONE;
-                r = sd_netlink_message_append_u8(req, IFLA_INET6_ADDR_GEN_MODE, ipv6ll_mode);
-                if (r < 0)
-                        return log_link_error_errno(link, r, "Could not append IFLA_INET6_ADDR_GEN_MODE: %m");
+                if (!link_is_bridge_port(link)) {
```

Comment:
This will only make networkd ignore the IPv6 part of LinkLocalAddressing=; However, the IPv6LL is actually autoconfigured by the kernel but not networkd, and by default (i.e. nothing like networkd or so change IFLA_INET6_ADDR_GEN_MODE to IN6_ADDR_GEN_MODE_NONE, it will be IN6_ADDR_GEN_MODE_EUI64 if the `stable_secret` sysctl is not set or IN6_ADDR_GEN_MODE_STABLE_PRIVACY if set.